addons/binding/org.openhab.binding.globalcache/src/main/java/org/openhab/binding/globalcache/GlobalCacheBindingConstants.java

```diff
@@ -63,6 +63,12 @@
     public static final String ACTIVE_CABLE_SERIAL = "FLEX_SERIAL";
     public static final String ACTIVE_CABLE_RELAY = "FLEX_RELAY";
 
+    // Serial readers
+    public static final String ENABLE_TWO_WAY_1 = "enableTwoWay1";
```

Comment:
Maybe use a few extra characters everywhere like `ENABLE_TWO_WAY_PORT_1` etc, it sounds more sensible than `ENABLE_TWO_WAY_1`. So add the fact that you are referring to a property for either port 1 or port 2.